apps/massa_proxy/lib/massa_proxy/runtime/grpc/protocol/action/protocol.ex

```diff
@@ -0,0 +1,67 @@
+defmodule MassaProxy.Runtime.Grpc.Protocol.Action.Protocol do
```

Comment:
The module name is kind of meh, easy to confuse it with something else. Suggestions for a better name?

Reply:
remove last Protocol or join with Action like ActionProtocol helps?